services/controllerhost/api_handlers.go

```diff
@@ -143,11 +150,20 @@ func areExtentStoresHealthy(context *Context, extent *m.DestinationExtent) bool
 	}
 
 	for _, h := range storeIDs {
+
+		status := "up"
```

Comment:
We can make this up, down and goingDown an enum as well and define it as a const the top.